<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="emblem-readonly"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 40, 49);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_add_new">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>450</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 62, 70);
border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="text_edit_website">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>310</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(34, 40, 49);
color: white;
padding-top: 3px;
padding-left: 3px;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_edit_password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(34, 40, 49);
color: white;
padding-top: 3px;
padding-left: 3px;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_edit_username">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>310</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(34, 40, 49);
color: white;
padding-top: 3px;
padding-left: 3px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_button">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    background-color: rgb(34, 40, 49);
	color: white;
	border: 2px solid rgb(150, 150, 150);
}
QPushButton:hover:!pressed {
   background-color: rgb(150, 150, 150);
}</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(238, 238, 238);
padding-top: 3px;
padding-left: 3px;</string>
     </property>
     <property name="text">
      <string>Website:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(238, 238, 238);
padding-top: 3px;
padding-left: 3px;</string>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(238, 238, 238);
padding-top: 3px;
padding-left: 3px;</string>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>10</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(238, 238, 238);
padding-top: 3px;
padding-left: 3px;</string>
     </property>
     <property name="text">
      <string>New Password Entry</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scroll_area">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>540</width>
      <height>440</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 62, 70);
border-radius: 10px;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>540</width>
       <height>440</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>70</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 62, 70);
border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="sign_out_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>125</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    background-color: rgb(34, 40, 49);
	border: 2px solid rgb(150, 150, 150);
}
QPushButton:hover:!pressed {
   background-color: rgb(150, 150, 150);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/assets/icons/log-out.svg</normaloff>:/assets/icons/log-out.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="change_mpassword_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>75</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    background-color: rgb(34, 40, 49);
	color: white;
	border: 2px solid rgb(150, 150, 150);
}
QPushButton:hover:!pressed {
   background-color: rgb(150, 150, 150);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/assets/icons/account.svg</normaloff>:/assets/icons/account.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>25</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    background-color: rgb(34, 40, 49);
	color: white;
	border: 2px solid rgb(150, 150, 150);
}
QPushButton:hover:!pressed {
   background-color: rgb(150, 150, 150);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/assets/icons/refresh.svg</normaloff>:/assets/icons/refresh.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
